### DISCUSSION TITLE
Testing docs

### DISCUSSION THREAD

**Discussion by dan-dazlab** [vueform/vueform#358] [Ideas]
It would be good to add examples for testing. Like create form with schema and mock it.
I just started to sorted out with this. It might be simple enough but it would be good to have a starting point in the docs.

**Comment by thely** [comment]
See #295. In the very least, Vueform does not work with vitest and vue/test-utils. Adam mentioned on Discord that the tests for the Vueform library were written with Jest, but no docs as of yet on testing an app with Jest.

**Comment by adamberecz** [comment]
Yeah—unfortunately I haven't got this yet. Maybe this could be used for inspiration:
https://github.com/vueform/vueform/tree/main/tests

(I believe @dan-dazlab already posted an example setup [here](https://github.com/vueform/vueform/issues/295#issuecomment-2480715442) 🙏)
